packages/components/src/server-side-render/index.js

```diff
@@ -20,10 +20,11 @@ import { addQueryArgs } from '@wordpress/url';
 import Placeholder from '../placeholder';
 import Spinner from '../spinner';
 
-export function rendererPathWithAttributes( block, attributes = null ) {
+export function rendererPath( block, attributes = null, urlQueryArgs = {} ) {
```

Comment:
Given this is an exported function, do we need to keep (or deprecate) the original `rendererPathWithAttributes()`?

Reply:
Phrased another way: maybe we should keep the name as `rendererPathWithAttributes()`, and it simply supports a third `urlQueryArgs` argument.

Reply:
Hi @danielbachhuber, the function rendererPathWithAttributes is exported just for testing but is not exported outside of @wordpress/components as we only export the default export outside of the module (the component) https://github.com/WordPress/gutenberg/blob/0c8481cc72c14499dead222a250d96355fdb4f03/packages/components/src/index.js#L51).
No plugin could have used this function so even if we change the name I don't think deprecations are necessary.

Reply:
Ah, makes sense. Thanks for clarifying @jorgefilipecosta